x/concentrated-liquidity/incentives.go

```diff
@@ -49,3 +49,20 @@ func (k Keeper) getUptimeAccumulators(ctx sdk.Context, poolId uint64) ([]accum.A
 
 	return accums, nil
 }
+
+// nolint: unused
+// getUptimeAccumulatorValues gets the accumulator values for the supported uptimes for the given poolId
+// Returns error if accumulator for the given poolId does not exist.
+func (k Keeper) getUptimeAccumulatorValues(ctx sdk.Context, poolId uint64) ([]sdk.DecCoins, error) {
```

Comment:
Note to reviewer: I ended up not needing to use this for this specific PR, but given that it is a well-tested and useful helper to have I suggest we don't delete it